pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -425,23 +865,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Method: Optimization\n",
-    "Gradient-based optimization with IPOPT with stochastic_program()\n",
+    "## Step 5: Method for Optimization\n",
+    "Gradient-based optimization with IPOPT using stochastic_program().\n",
     "\n",
-    "This function solves twice: It solves the square version of the MBDoE problem first, and then unfixes the design variables as degrees of freedom and solves again. In this way the optimization problem can be well initialized. "
+    "This function solves twice:\n",
+    "1. Solves the square version of the MBDoE problem first,\n",
+    "    * We are fixing all of the degrees of freedom for the temperatures. Recall temperature is constant but is allowed to vary between time steps.\n",
```

Comment:
"We first fix the experiment design decisions and solve the simulation problem (zero degrees of freedom). This facilitates initialization."